src/sentry/processing/realtime_metrics/base.py

```diff
@@ -97,6 +98,14 @@ def get_lpq_projects(self) -> Set[int]:
         """
         raise NotImplementedError
 
+    def is_lpq_project(self, project_id: int) -> bool:
+        """
+        Checks whether the given project is currently using the low priority queue.
+
+        Returns a bool.
```

Comment:
nit: i think the line here and in `redis.py` could be removed since the types already declare this, and it's fairly obvious what the return value represents, making the method and signature fairly self-describing. in other words, i feel this doesn't add in additional information that can't be easily derived from the method name or the return value.